instrumentation/android-instrumentation/src/main/java/io/opentelemetry/android/instrumentation/AndroidInstrumentationLoader.kt

```diff
@@ -41,7 +41,7 @@ interface AndroidInstrumentationLoader {
          * Convenience method for [AndroidInstrumentationLoader.getByType].
          */
         @JvmStatic
-        fun <T : AndroidInstrumentation> getInstrumentation(type: Class<out T>): T? = get().getByType(type)
+        fun <T : AndroidInstrumentation> getInstrumentation(type: Class<out T>): T = get().getByType(type)!!
```

Comment:
I tweaked this to force non-null types and inlined the version in `OpenTelemetryRumInitializer`, given that every call in there was already doing this.